src/aca_main.cpp

```diff
@@ -71,7 +80,7 @@ std::atomic_ulong g_total_vpcs_table_mutex_time(0);
 std::atomic_ulong g_total_update_GS_time(0);
 
 bool g_demo_mode = false;
-bool g_debug_mode = false;
+bool g_debug_mode = true;
```

Comment:
Are we going to keep debug mode true?

Reply:
No, it was only for printing debug logs. Reverted. Good catch, thanks.